sw/device/silicon_creator/lib/drivers/flash_ctrl.c

```diff
@@ -27,6 +27,9 @@ void flash_ctrl_init(void) {
   // Initialize the flash controller.
   abs_mmio_write32(kBase + FLASH_CTRL_INIT_REG_OFFSET,
                    bitfield_bit32_write(0u, FLASH_CTRL_INIT_VAL_BIT, true));
+
+  // Enable flash execution
```

Comment:
Can you add a TODO here? This should be moved into a separate function so that we can enable execution only after the code partition has been verified. Thanks!

Reply:
i was going to file a separate issue.